api/clients/disperser_client.go

```diff
@@ -107,6 +107,28 @@ func (c *disperserClient) DisperseBlob(ctx context.Context, data []byte, quorums
 }
 
 func (c *disperserClient) DisperseBlobAuthenticated(ctx context.Context, data []byte, quorums []uint8) (*disperser.BlobStatus, []byte, error) {
+	// first check if signer is valid
+	accountId, err := c.signer.GetAccountID()
+	if err != nil {
+		return nil, nil, err
```

Comment:
would it be even more clear if we add something like `please configure signer key if you want to use authenticated endpoint`?